api/types.proto

```diff
@@ -299,20 +316,27 @@ message UpdateStatus {
 		UPDATING = 1;
 		PAUSED = 2;
 		COMPLETED = 3;
-		// TODO(aaronl): add ROLLING_BACK, ROLLED_BACK as part of
-		// rollback support.
+		ROLLING_BACK = 4;
+		ROLLBACK_PAUSED = 5; // if a rollback fails
+		ROLLBACK_COMPLETED = 6;
 	}
 
 	// State is the state of this update. It indicates whether the
-	// update is in progress, completed, or is paused.
+	// update is in progress, completed, paused, rolling back, or
+	// finished rolling back.
 	UpdateState state = 1;
 
 	// StartedAt is the time at which the update was started.
 	Timestamp started_at = 2;
 
-	// CompletedAt is the time at which the update completed.
+	// CompletedAt is the time at which the update completed successfully,
+	// paused, or finished rolling back.
 	Timestamp completed_at = 3;
 
+	// StartedRollingBackAt is the time at which the update started rolling
+	// back to the previous version.
+	Timestamp started_rolling_back_at = 5;
+
```

Comment:
Can we include a progress message in here to support `running/replicas` in command output? That will let us get rid of some extra queries to task sets and make that  more accurate.